tests/b2share_functional_tests/cli_tests/test_schema_cli.py

```diff
@@ -59,6 +59,10 @@
     "required": ["test_field1"]
 }
 
+configurations = [({'config': {'PREFERRED_URL_SCHEME': 'https'}}),
```

Comment:
Do we have a reason to test for both schemes? because it takes more time to run the tests and we should always run over https (we send confidential information). So I'd remove these added lines completely.